src/lib/pages/interaction/component/common/EmptyState.tsx

```diff
@@ -41,3 +42,7 @@ export const ModuleEmptyState = ({
     </Flex>
   );
 };
+
+export const NoImageEmptyState = ({ desc }: { desc: string }) => (
+  <ModuleEmptyState h="fit-content" p={4} hasImage={false} description={desc} />
+);
```

Comment:
why don't we use these NoImage params as a default of params of `ModuleEmptyState`? Only `ModuleEmptyState` is called directly just only 2 times. Also remove default description from `ModuleEmptyState`. Only one location use that default description